app/modules/notifications/models.py

```diff
@@ -217,16 +218,19 @@ def set_individual_merge(self, individuals, encounters, request_data):
         for indiv in individuals:
             ind_data = {'guid': indiv.guid, 'primaryName': indiv.get_primary_name()}
             self.data['individual_list'].append(ind_data)
+            source_names.append(indiv.get_primary_name())
             if 'target_individual_guid' not in self.data:
                 self.data['target_individual_guid'] = str(indiv.guid)
                 self.data['target_individual_name'] = indiv.get_primary_name()
-            else:
-                source_names.append(indiv.get_primary_name())
+
```

Comment:
These target fields are arguably redundant and according to slack, should not be displayed to the user. Chose to keep them in to respect the frontend freeze/avoid possibly mucking up existing FE work.